.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "java.compile.nullAnalysis.mode": "automatic"
```

Comment:
why is this added to the code base? 👎

Reply:
Not sure when this was added but I've removed it.

Reply:
all fine!